src/Stripe.net/Services/Topups/StripeTopupService.cs

```diff
@@ -15,35 +15,28 @@ public StripeTopupService(string apiKey = null)
 
         public bool ExpandSource { get; set; }
 
-        public virtual StripeTopup Create(StripeTopupCreateOptions createOptions, StripeRequestOptions requestOptions = null)
+        public virtual StripeTopup Cancel(string topupId, StripeRequestOptions requestOptions = null)
         {
             return Mapper<StripeTopup>.MapFromJson(
                 Requestor.PostString(
-                    this.ApplyAllParameters(createOptions, Urls.Topups, false),
+                    this.ApplyAllParameters(null, $"{Urls.Topups}/{topupId}/cancel", false),
                     this.SetupRequestOptions(requestOptions)));
```

Comment:
In other services like the Payout one we did
```
return this.Post($"{Urls.BaseUrl}/payouts/{payoutId}/cancel", requestOptions);
```
which is  bit more readable, Not sure what the difference is between `StripeService` and `StripeBasicService` though

Reply:
Ack. I'll leave this one as-is for now. Let's add this to the list of things that should be standardized throughout the library.